configs/roles/insights.json

```diff
@@ -2,13 +2,14 @@
   "roles": [
     {
       "name": "Insights administrator",
-      "description": "Perform any available operation against any Insights resource.",
+      "display_name": "Advisor administrator",
+      "description": "Perform any available operation against any advisor resource.",
       "system": true,
       "platform_default": true,
-      "version": 6,
+      "version": 7,
       "access": [
         {
-          "permission": "insights:*:*"
+          "permission": "advisor:*:*"
```

Comment:
Should we modify the one in the permissions folder? We might have to think of a way to delete the old permission "insights:*:*"

Reply:
Yep, you're right. I just realized that looking at another PR. For the short term, I'm thinking of just adding an `advisor.json` file in the permissions directory.